rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -1043,10 +1043,25 @@ void spu_recompiler::IRET(spu_opcode_t op)
 
 void spu_recompiler::BISLED(spu_opcode_t op)
 {
-	fmt::throw_exception("Unimplemented instruction" HERE);
+	extern asmjit::X86Mem t ;
+	c->mov(t, SPU_OFF_32(gpr, op.ra, &v128::_u32, 3));
+	c->and_(t, 0x3fffc);
+	
+	const XmmLink& vr = XmmAlloc();
+	c->movdqa(vr, XmmConst(_mm_set_epi32(spu_branch_target(m_pos + 4), 0, 0, 0)));
+	c->movdqa(SPU_OFF_128(gpr, op.rt), vr);
+	//todo:
+	//if (/*external event*/){
+	//if (op.d || op.e) c->or_(*addr, op.e << 26 | op.d << 27); // interrupt flags stored to PC
+	// c->mov(SPU_OFF_32(pc), t);}
+	// else
+	c->mov(SPU_OFF_32(pc), SPU_OFF_32(vr));
+	c->unuse(vr);
+	fmt::throw_exception("TODO: SPU external event check" HERE);
```

Comment:
Change these trow to TODO!

Reply:
Instead of "throw_exeption"?

Reply:
Yes. If you trow exception it will mean critical error happened here and this program should't work anymore. Do you know how to do it?